poetry.lock

```diff
@@ -65,17 +65,17 @@ python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
 
 [[package]]
 name = "attrs"
-version = "21.2.0"
```

Comment:
I think you don't need to touch the lockfile, you haven't changed dependencies. Can you revert this?

Reply:
okok